### DISCUSSION TITLE
Parsing experiences in a resume

### DISCUSSION THREAD

**Discussion by nbhector** [explosion/spaCy#9036] [Help: Coding & Implementations] [answered]
Hello,

I have used Spacy to parse the basic information from someone's resume such as name, skills, email, and phone number.  However, extracting the experience is the most difficult to extract since it doesn't have a specific entity / label.  I assume that training the nlp pipeline with a dataset that has experiences labeled would be a solution, but I wanted to hear from others on what I can do and how Spacy would assist me in extracting the experiences.

**Comment by thomashacker** [comment, marked as answer]
Hello, 
I think it's a good approach to look at some example experiences that you'd like to analyze and write down all the information you want to extract. You can try to use a pre-trained NER and see how well it performs with its trained labels ( e.g. extracting company names, how long they worked there). But of course, you can also create your own labeled dataset which is more domain-specific, and train a new model.
